csharp/PublicAPI/net471/PublicAPI.Shipped.txt

```diff
@@ -678,7 +678,8 @@ ParquetSharp.Schema.GroupNode.FieldCount.get -> int
 ParquetSharp.Schema.GroupNode.FieldIndex(ParquetSharp.Schema.Node! node) -> int
 ParquetSharp.Schema.GroupNode.FieldIndex(string! name) -> int
 ParquetSharp.Schema.GroupNode.Fields.get -> ParquetSharp.Schema.Node![]!
-ParquetSharp.Schema.GroupNode.GroupNode(string! name, ParquetSharp.Repetition repetition, System.Collections.Generic.IReadOnlyList<ParquetSharp.Schema.Node!>! fields, ParquetSharp.LogicalType? logicalType = null) -> void
+ParquetSharp.Schema.GroupNode.GroupNode(string! name, ParquetSharp.Repetition repetition, System.Collections.Generic.IReadOnlyList<ParquetSharp.Schema.Node!>! fields, ParquetSharp.LogicalType? logicalType) -> void
+ParquetSharp.Schema.GroupNode.GroupNode(string! name, ParquetSharp.Repetition repetition, System.Collections.Generic.IReadOnlyList<ParquetSharp.Schema.Node!>! fields, ParquetSharp.LogicalType? logicalType = null, int fieldId = -1) -> void
```

Comment:
:+1: I was initially worried this was a breaking change, but it looks like this should maintain source and binary compatibility as the default value is part of the source API but not the ABI. For my own reference, there are docs on adding optional parameters at https://github.com/dotnet/roslyn/blob/main/docs/Adding%20Optional%20Parameters%20in%20Public%20API.md